src/components/custom/_signup-public.js

```diff
@@ -171,16 +171,17 @@ const StyledText = styled(Text)`
     }
 `
 
-const StyledSpan = styled.span`
-    font-size: 13px;
-    font-weight: 500;
-    color: var(--color-grey-13);
-    margin-left: 4px;
+// const StyledSpan = styled.span`
+//     font-size: 13px;
+//     font-weight: 500;
+//     color: var(--color-grey-13);
+//     margin-left: 4px;
+
+//     @media ${device.tabletL} {
+//         margin-left: 8px;
+//     }
+// `
```

Comment:
```suggestion
```
Any reasons leaving this code uncommented instead of deleted?

Reply:
Still WIP actually...haven't gotten confirmation from design if this is okay. If it is then i'll take this off 😂